README.md

```diff
@@ -129,25 +158,27 @@ parameter_example.test:
     'roles': [
         { 'role': '$TEST_ROLE',
           $FIELD: $VALUE,
-          ‘my_number’: $my_number,
-          ‘my_boolean’: $my_boolean,
-          ‘str_no_quotes’: ‘$str_no_quotes’,
-          ‘str_with_quotes’: $str_with_quotes,
+          'my_number': $my_number,
+          'my_boolean': $my_boolean,
+          'str_no_quotes': '$str_no_quotes',
+          'str_with_quotes': $str_with_quotes,
+          'str_with_spaces': $str_with_spaces,
         },
     ],
 
-    'test': [‘$TEST_ROLE’],
+    'test': ['$TEST_ROLE'],
 }
 
-$ python test_driver.py parameter_example.test --manifest-args=”\
+$ python test_driver.py parameter_example.test --manifest-args \
     STL_FILE=example.stl \
     TEST_ROLE=example::rReceiver \
-    FIELD=’new_field’ \
-    VALUE=’new_value’ \
+    FIELD=\'new_field\' \
+    VALUE=\'new_value\' \
     my_number=3 \
     my_boolean=True \
     str_no_quotes=World \
-    str_with_quotes=’bar’”
+    str_with_quotes=\'bar\' \
+    str_with_spaces="'foo bar baz'"
```

Comment:
The shell will evaluate the argument as the following:

str_with_spaces='foo bar baz'

You could wrap the whole argument in quotes as well, as:
"str_with_spaces='foo bar baz'"

We can't escape the inner single quotes since the spaces will separate on the command line.